c7n/resources/sfn.py

```diff
@@ -40,6 +41,59 @@ class resource_type(TypeInfo):
     }
 
 
+class DescribeActivity(DescribeSource):
+
+    def augment(self, resources):
+        resources = super().augment(resources)
+        return universal_augment(self.manager, resources)
+
+
+@resources.register('activity')
```

Comment:
The name of the resource here lacks context. Would it make sense to rename it to something that contains sfn?

Reply:
renaming to sfn-activity